extensions/data-workspace/src/dialogs/newProjectQuickpick.ts

```diff
@@ -87,5 +89,53 @@ export async function createNewProjectWithQuickpick(workspaceService: WorkspaceS
 		continue;
 	}
 
-	await workspaceService.createProject(projectName, vscode.Uri.file(projectLocation), projectType.id, undefined);
+	let sdkStyle;
+	if (projectType.sdkOption) {
+		// 4. SDK-style project or not
+		const sdkLearnMoreButton: vscode.QuickInputButton = {
+			iconPath: new vscode.ThemeIcon('link-external'),
+			tooltip: constants.LearnMore
+		};
+		const quickPick = vscode.window.createQuickPick();
+		quickPick.items = [{ label: constants.Yes }, { label: constants.No }];
```

Comment:
You can just pass the array of strings, no need to create objects

Reply:
there's an error that string is unassignable to type QuickPickItem when I try that :( looks like you can only pass in an array of strings when doing vscode.window.showQuickPick(), but not when setting the items after the quickpick has been created.